packages/xchain-thorchain/src/client.ts

```diff
@@ -242,36 +216,6 @@ class Client implements ThorchainClient, XChainClient {
     return getExplorerTxUrl({ urls: this.explorerUrls, network: this.network, txID })
   }
 
-  /**
-   * Set/update a new phrase
-   *
-   * @param {string} phrase A new phrase.
-   * @returns {Address} The address from the given phrase
-   *
-   * @throws {"Invalid phrase"}
-   * Thrown if the given phase is invalid.
-   */
-  setPhrase(phrase: string, walletIndex = 0): Address {
```

Comment:
Why has `setPhrase` been removed?

Reply:
it's removed because this class now extends the BaseXChainClient, which implements setPhrase